content/minfraud/normalizing-email-addresses-for-minfraud.mdx

```diff
@@ -59,9 +59,9 @@ yourself. Below are the steps to take to do this.
 You can review the code in our client APIs see how to normalize an email address
 in various languages.
 
-- [.NET (C#)](https://github.com/maxmind/minfraud-api-dotnet/blob/main/MaxMind.MinFraud/Request/Email.cs#L150)
-- [Java](https://github.com/maxmind/minfraud-api-java/blob/main/src/main/java/com/maxmind/minfraud/request/Email.java#L169)
-- [JavaScript](https://github.com/maxmind/minfraud-api-node/blob/main/src/request/email.ts#L75)
-- [PHP](https://github.com/maxmind/minfraud-api-php/blob/main/src/MinFraud/Util.php#L34)
-- [Python](https://github.com/maxmind/minfraud-api-python/blob/main/minfraud/request.py#L83)
+- [.NET (C#)](https://github.com/maxmind/minfraud-api-dotnet/blob/main/MaxMind.MinFraud/Request/Email.cs#L395)
+- [Java](https://github.com/maxmind/minfraud-api-java/blob/main/src/main/java/com/maxmind/minfraud/request/Email.java#L414)
+- [JavaScript](https://github.com/maxmind/minfraud-api-node/blob/main/src/request/email.ts#L313)
+- [PHP](https://github.com/maxmind/minfraud-api-php/blob/main/src/MinFraud/Util.php#L309)
```

Comment:
For this one, should we link to line 284 instead? That would be to the email cleaning.